test/runner/command.js

```diff
@@ -59,15 +59,23 @@ const argv = yargs( process.argv.slice( 2 ) )
 			"Leave the browser open for debugging. Cannot be used with --headless.",
 		conflicts: [ "headless" ]
 	} )
+	.option( "retries", {
+		alias: "r",
+		type: "number",
+		description: "Number of times to retry failed tests by refreshing the URL."
+	} )
```

Comment:
We can, but the change of order seemed logical to me. `retries` and `hard-retries` no longer only apply to `browserstack`, and I moved `run-id` below `browserstack` because it *does* only apply to browserstack.